node/src/types/chainspec/accounts_config/delegator_config.rs

```diff
@@ -9,12 +9,12 @@ use casper_types::{
     PublicKey,
 };
 #[cfg(test)]
-use casper_types::{SecretKey, U512};
+use casper_types::{AsymmetricType, SecretKey, U512};
 
 #[cfg(test)]
 use crate::testing::TestRng;
 
-#[derive(PartialEq, Ord, PartialOrd, Eq, Serialize, Deserialize, DataSize, Debug, Copy, Clone)]
```

Comment:
Why is it not `Copy` anymore?

Reply:
`k256::ecdsa::Signature` doesn't support `Copy` so we can't.